src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectReportsController.kt

```diff
@@ -203,6 +206,20 @@ class ProjectReportsController(
         configs.map { ExistingAcceleratorReportConfigPayload(it) })
   }
 
+  @ApiResponse200
+  @ApiResponse400
+  @ApiResponse404
+  @PostMapping("/configs")
+  @Operation(summary = "Update accelerator report configuration.")
```

Comment:
This summary matches the one below it even though they have slightly different parameters (projectId vs configId). Might make sense to explain why they're different, if nothing else then for devs coming back to this code after a long time